spec/decorators/absence_request_decorator_spec.rb

```diff
@@ -0,0 +1,122 @@
+require "rails_helper"
+
+RSpec.describe AbsenceRequestDecorator, type: :model do
+  subject(:absence_request_decorator) { described_class.new(absence_request) }
+  let(:absence_request) { FactoryBot.create(:absence_request, absence_type: :vacation) }
+
+  describe "attributes relevant to AbsenceRequest" do
+    it { is_expected.to respond_to :absence_type }
+    it { is_expected.to respond_to :end_date }
+    it { is_expected.to respond_to :id }
+    it { is_expected.to respond_to :request_type }
+    it { is_expected.to respond_to :status }
+    it { is_expected.to respond_to :start_date }
+    it { is_expected.to respond_to :to_model }
+  end
+
+  describe "#absence_type_icon" do
+    context "when absence_type is vacation" do
+      it "returns vacation icon" do
+        expect(absence_request_decorator.absence_type_icon).to eq "lux-icon-vacation"
+      end
+    end
+    context "when absence_type is sick" do
+      let(:absence_request) { FactoryBot.create(:absence_request, absence_type: :sick) }
+      it "returns sick icon" do
+        expect(absence_request_decorator.absence_type_icon).to eq "lux-icon-hospital"
```

Comment:
do we want more of these tests? For title I wrote them all; not sure if we want to do that here, also

Reply:
Sure we can have more tests.